examples/Cargo.toml

```diff
@@ -18,10 +16,8 @@ members = [
     "meta-counter",
     "native-fungible",
     "non-fungible",
-    "publish-read-data-blob",
     "rfq",
     "social",
-    "track-instantiation",
```

Comment:
Not sure if they should be removed from the workspace. At least we want to know if they fail to build due to some change in their dependencies.

Reply:
But, isn't it the code tested in
```github
  linera-sdk-tests-fixtures:
    needs: changed-files
    runs-on: ubuntu-latest
    timeout-minutes: 10

    steps:
    - uses: actions/checkout@v4
    - uses: actions-rust-lang/setup-rust-toolchain@v1
    - name: Install Protoc
      uses: arduino/setup-protoc@v3
      with:
        repo-token: ${{ secrets.GITHUB_TOKEN }}
    - name: Run linera-sdk fixtures
      run: |
        cd linera-sdk/tests/fixtures
        cargo test --locked
    - name: Run linera-sdk tests application lints
      run: |
        cd linera-sdk/tests/fixtures
        cargo fmt -- --check
        cargo clippy --all-targets --all-features --target wasm32-unknown-unknown --locked
```

So, I think it is still being tested.

Reply:
Makes sense